### PR TITLE
LPS-170252 Update case results with issues

### PR DESCRIPTION
**Pull request by rafaelfs98**
[LPS-170252](https://issues.liferay.com/browse/LPS-170252)

### CODE REVIEW

**Comment by kevenleone** [comment]
@rafaelfs98 can you rename this commit, because `LPS added` is weird

**Comment by rafaelfs98** [comment]
@kevenleone  it was good like this

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/Subtask/SubtaskCompleteModal.tsx [hunk 1]
```diff
@@ -42,35 +45,42 @@ type SubTaskCompleteModalProps = {
...
+
+	const subtaskIssues = data?.items || [];
 
 	const issues = subtaskIssues
 		.map((subtaskIssue: TestraySubTaskIssue) => subtaskIssue?.issue?.name)
 		.join(', ');
 
 	const {
 		formState: {errors},
+
```
Comment (welitonpe) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/Subtask/SubtaskCompleteModal.tsx:77]: Remove this blank space.
```suggestion

```

**Review by samuelleolopes — APPROVED** [review]
LGTM :fire:

**Comment by rafaelfs98** [comment]
@kevenleone , @welitonpe resolved suggestions

**Comment by kevenleone** [comment]
ci:test:sf

**Review by kevenleone — APPROVED** [review]
LGTM 🔥

**Comment by kevenleone** [comment]
ci:forward

**Comment by kevenleone** [comment]
ci:stop

**Pull request closed without merging**